src/main/java/io/reactiverse/pgclient/PgConnectOptions.java

```diff
@@ -452,7 +434,10 @@ public int hashCode() {
     result = 31 * result + password.hashCode();
     result = 31 * result + (cachePreparedStatements ? 1 : 0);
     result = 31 * result + pipeliningLimit;
-    result = 31 * result + (domainSocket ? 1 : 0);
     return result;
   }
+
+  public boolean isUsingDomainSocket() {
```

Comment:
Using this public method for letting https://github.com/reactiverse/reactive-pg-client/blob/6920ec7489b63dfe359e65eb57d226ecba01ac25/src/main/java/io/reactiverse/pgclient/PgClient.java#L72 to know whether a vertx instance should `setPreferNativeTransport()`